test/tests.js

```diff
@@ -1660,6 +1660,24 @@ describe("passport-saml /", function () {
       metadata.should.containEql(samlConfig.logoutCallbackUrl);
     });
 
+    it("generateServiceProviderMetadata contains WantAssertionsSigned", function () {
```

Comment:
We will also need a test to verify that a bad assertion signature is caught.

Reply:
Bad assertion signature is now tested in these tests:

https://github.com/node-saml/passport-saml/blob/f6bce345fe6d7af58a71d246290ae9d1b2167227/test/test-signatures.spec.ts#L118-L119

https://github.com/node-saml/passport-saml/blob/f6bce345fe6d7af58a71d246290ae9d1b2167227/test/test-signatures.spec.ts#L130-L131